src/components/containers/ConnectedMellowLpDepositForm/ConnectedMellowLpDepositForm.tsx

```diff
@@ -0,0 +1,221 @@
+import { AugmentedMellowLpVault } from "@utilities";
+import { routes } from '@routes';
+import { useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import MellowLpDepositForm from "src/components/interface/MellowLpDepositForm/MellowLpDepositForm";
+import { Panel } from "src/components/atomic";
+import React, { useState, useEffect, ReactNode } from "react";
+import { colors } from '@theme';
+import { Box } from "@mui/system";
+
+
+export type ConnectedMellowLpDepositFormProps = {
+    vault: AugmentedMellowLpVault;
+    onReset: () => void;
+}
+
+export enum DepositStates {
+    INITIALISING = "INITIALISING",
+    PROVIDER_ERROR = "PROVIDER_ERROR",
+    APPROVE_REQUIRED = "APPROVE_REQUIRED",
+    APPROVING = "APPROVING",
+    APPROVE_FAILED = "APPROVE_FAILED",
+    APPROVED = "APPROVED",
+    DEPOSITING = "DEPOSITING",
+    DEPOSIT_FAILED = "DEPOSIT_FAILED",
+    DEPOSIT_DONE = "DEPOSIT_DONE",
+};
+
+type TextProps = {
+    bold?: boolean;
+    children?: ReactNode;
+    green?: boolean;
+    red?: boolean;
+};
+const Text = ({ bold, children, green, red }: TextProps) => (
+    <Box component='span' sx={{
```

Comment:
just use span... I don't see any benefits of using Box with component="span". Use span + style

Reply:
Resolved. https://github.com/Voltz-Protocol/voltz-ui/pull/295/commits/a44c3c48e2a2660e1fc14f6eeeda646152d2edea